recipes/ninja/1.10.x/conanfile.py

```diff
@@ -0,0 +1,51 @@
+from conans import ConanFile, CMake, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+class NinjaConan(ConanFile):
+    name = "ninja"
+    description = "Ninja is a small build system with a focus on speed"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/ninja-build/ninja"
+    settings = "os", "arch", "compiler"
+    exports_sources = "CMakeLists.txt"
+    generators = "cmake"
+
+    _source_subfolder = "source_subfolder"
+    _cmake = None
+
+    def configure(self):
+        if self.settings.compiler == "Visual Studio":
+            if self.settings.compiler.runtime != "MT":
+                raise ConanInvalidConfiguration("Only MT MSVC runtime is supported")
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+
+        self._cmake = CMake(self)
+        self._cmake.configure()
+        return self._cmake
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        os.rename("ninja-%s" % self.version, self._source_subfolder)
+
+    def build(self):
+        cmake = self._configure_cmake()
+        cmake.build()
+
+    def package(self):
+        self.copy(pattern="COPYING", dst="licenses",
+                  src=self._source_subfolder)
+        cmake = self._configure_cmake()
+        cmake.install()
```

Comment:
there is no rule for install on that cmake file. Actually it could be improved, like making all test optional when building, it would save some time.

Reply:
Just realized that.  I was looking at their master instead of the 1.10 tag.